tests/functional/Codeception/Docker.php

```diff
@@ -213,18 +250,47 @@ public function grabFileContent(string $source, string $container = self::DEPLOY
      * @throws \Robo\Exception\TaskException
      */
     public function runEceToolsCommand(string $command, string $container, array $variables = []): bool
+    {
+        return $this->taskBash($container)
+            ->envVars($this->prepareVariables($variables))
+            ->exec(sprintf('php %s/bin/ece-tools %s', $this->_getConfig('system_ece_tools_dir'), $command))
+            ->run()
+            ->wasSuccessful();
+    }
+
+    /**
+     * Runs bin/magento command on Docker container
+     *
+     * @param string $command
+     * @param string $container
+     * @param array $variables
+     * @return bool
+     * @throws \Robo\Exception\TaskException
+     */
+    public function runBinMagentoCommand(string $command, string $container, array $variables = []): bool
+    {
+        return $this->taskBash($container)
+            ->envVars($this->prepareVariables($variables))
+            ->exec(sprintf('php %s/bin/magento %s', $this->_getConfig('system_magento_dir'), $command))
+            ->run()
```

Comment:
On the Magecat backlog, I want to add `->printOutput(false)` to just about all the Robo tasks so they don't blow up the console as much. If we need to log or assert against the output of those tasks, we can use `->getOutputData()` after the task has been run.

It'd probably be nice to head this off in ECE Tools so we don't run into the same problem here.